db/migrate/20180215094320_create_join_table_actors_movies.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableActorsMovies < ActiveRecord::Migration[5.1]
+  def change
+    create_join_table :actors, :movies do |t|
+      # t.index [:actor_id, :movie_id]
```

Comment:
why do we have these comments?

Reply:
These are added by default when we create this join table migration.